### DISCUSSION TITLE
Does opentrack need x86 or can it run on a beefy ARM platform ?

### DISCUSSION THREAD

**Discussion by Don-Moahskarton** [opentrack/opentrack#1456] [Q&A]
The title says it all.
I'm wondering if I could run opentrack on an embedded device like the Nvidia Jetson TX2 (Quad-core ARM Cortex A-57 - see https://developer.nvidia.com/embedded/jetson-tx2)

I could not see any requirement mentionned for x86 platform. Does anyone see a major roadblock, or is it more like "never tried, but should not be a problem".

**Comment by sthalik** [comment]
It might already run on non-x86 at least for Linux. Please report your results, including any problems you run into.
